src/Microsoft.TemplateEngine.IDE/Bootstrapper.cs

```diff
@@ -88,41 +88,41 @@ public async Task<ICreationEffects> GetCreationEffectsAsync(ITemplateInfo info,
         #region Template Package Management
 
         /// <summary>
-        /// Get the list of available template packages
+        /// Gets the list of available templates packages.
         /// </summary>
         /// <param name="cancellationToken"></param>
-        /// <returns>the list of the template packages</returns>
-        public Task<IReadOnlyList<ITemplatesSource>> GetTemplatesSources(CancellationToken cancellationToken = default)
+        /// <returns>the list of the templates packages.</returns>
```

Comment:
This plural format looks wrong to me in any example I see. I say we revert this and go with "template packages".

Reply:
That's a confusion that I have with `TemplatesPackage`. How to do plural from it?

Reply:
Reverted to use "template package(s)"